core/domain/feedback_jobs_one_off.py

```diff
@@ -66,3 +66,34 @@ def reduce(key, stringified_message_ids):
                 'next_message_id': next_message_id,
                 'message_count': len(message_ids)
             })
+
+
+class FeedbackSubjectOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for updating the feedback subject."""
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [feedback_models.FeedbackThreadModel]
+
+    @staticmethod
+    def map(item):
+        if item.subject != '(Feedback from a learner)':
+            return
+        if not item.summary:
+            return
+        characterCount = 50
```

Comment:
Use UPPER_CASE for constants and consider making this a class-level constant.

We do not use camelCase for variable/constant names in Python. See the style rules.

EDIT: see also later comment re assets/constants.js

Reply:
Done